src/d_array.c

```diff
@@ -708,7 +708,7 @@ static t_int *tabsend_perform(t_int *w)
 {
     t_tabsend *x = (t_tabsend *)(w[1]);
     t_sample *in = (t_sample *)(w[2]);
-    int n = w[3];
+    t_int n = w[3];
```

Comment:
i think this is wrong.
`t_int` is practically never used for *int* values; it's main purpose seems to be to a way to provide "void-pointer arithmetic" in the dsp-graph.

(i think the use of `t_int` as return-type for `atom_getint` resp `atom_getintarg` is accidental and should be fixed)

Reply:
Gotcha. t_int sounded like a generic integer type, so I was preferring that in places where I thought it made sense.

Reply:
>(i think the use of t_int as return-type for atom_getint resp atom_getintarg is accidental and should be fixed)

Should I rectify this? It would solve a good number of warnings.

Reply:
yes, it should read `int n = (int)w[3];`